flexget/plugins/sites/piratebay.py

```diff
@@ -15,11 +15,7 @@
 
 log = logging.getLogger('piratebay')
 
-CUR_TLD = 'org'
-TLDS = 'com|org|sx|ac|pe|gy|to|se|gd|vg|%s' % CUR_TLD
```

Comment:
What about the old TLDs? We used to match them all and rewrite to the CUR_TLD.

Reply:
This is not applicable anymore, because the piratebay (as far as I know) does not provide domain-specific URLs anymore for torrents (only magnet links) and the only rewriting happening is to match the domain of the reverse-proxy (or the original website) that is used to access tpb